doc: Add LTE user guide (#13521)
nrfconnect/sdk-nrf: doc/nrf/protocols/lte/index.rst

For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_ , which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor’s nRF91 Series device.

The :ref:`nrf9160_ug_network_mode` section provides information about switching between LTE-M and NB-IoT.

Cellular IoT in |NCS|
*********************

The following sections explain the different applications, libraries, and samples that use the cellular IoT products.

Modem library
=============

.. include:: ../../device_guides/working_with_nrf/nrf91/nrf91_features.rst
    :start-after: nrf91_modem_lib_start
    :end-before: nrf91_modem_lib_end

Library Support
===============

The following nRF Connect SDK libraries are used for cellular IoT:

* :ref:`lib_modem`
* :ref:`lib_networking`
* :ref:`liblwm2m_carrier_readme` library
* :ref:`mod_memfault` library

Applications and samples
========================

The following applications use the cellular IoT in |NCS|:

* :ref:`asset_tracker_v2`
* :ref:`serial_lte_modem`
* :ref:`connectivity_bridge`
* `nRF Asset Tracker project`_

> The following samples are used for the cellular IoT development in |NCS|:

Reviewer: The current approach looks good to me.

* :ref:`memfault_sample` sample

Integrations
============

The following Integrations are available for cellular IoT in |NCS|:

* :ref:`ug_nrf_cloud` - `nRF Cloud`_ is `Nordic Semiconductor's IoT cloud platform`_ that allows you to remotely manage and update your IoT devices using :term:`Firmware Over-the-Air (FOTA) <Firmware Over-the-Air (FOTA) update>`.
* :ref:`ug_memfault`- A cloud-based web application with |NCS| compatibility that monitors devices and allows you to debug issues remotely through an LTE or Wi-Fi network.
* :ref:`ug_avsystem` - Software solution provider that provides a device management platform.

Power optimization
==================

For optimizing the power consumption of your cellular application, you can use the `Online Power Profiler (OPP)`_ tool.
See :ref:`app_power_opt_nRF91` for more information.

References
**********

* `nWP044 - Best practices for cellular IoT development`_ - White paper that describes the guidelines that you need to consider when you start the development for a low power cellular IoT product.
* `Cellular IoT asset tracking`_

> * `Accelerate Cellular Product Development`_

Author: @martiles : Please confirm are these 2 links required as part of the LTE guide?
* https://youtu.be/b1YW1J4aUps
* https://youtu.be/V-WlGbyRDZg

Please let me know if any other references need to be added.

Reviewer 2: Not required, but nice to have

.. _ug_lte:

LTE
###

Long Term Evolution (LTE) technology enables devices to be connected to a base station without an intermediate device or gateway.
LTE  is the fourth-generation (4G) wireless broadband standard built on existing technologies, replacing 3G.

> LTE maximizes bandwidth (BW) utilization and supports up to 300 Mbps.

Reviewer 3: ```suggestion
LTE maximizes bandwidth utilization and supports data rates up to 300 Mbps.
```

It is designed for indoor coverage, low cost, and long battery life.
NB-IoT operates in a small radio frequency band of 200 kHz.
It is ideal for applications like smart metering, environmental monitoring, and smart lighting.

LTE-M and NB-IoT are part of 5G.

nRF91 Series devices such as nRF9161, nRF9160, and Thingy:91 support both LTE-M and NB-IoT technologies.

Comparison between LTE-M and NB-IoT
===================================

The following table shows a comparison between LTE-M and NB-IoT in terms of advantages in using any of these standards:

+------------------+--------------------------------+--------------------------------+
|                  | LTE-M                          | NB-IoT                         |
+==================+================================+================================+
|                  | Higher throughput              | Longer range                   |
+                  +--------------------------------+--------------------------------+
|                  | Lower latency                  | Better penetration             |
+                  +--------------------------------+--------------------------------+

> |   Strengths      | Roaming                        | Power efficient at lower data  |

Reviewer 3: Why is roaming a strength of LTE-M but not of NB-IoT? Does this have something to do with operator offerings?

Reviewer: The link points to _nRF Connect for Desktop_ documentation front page, which probably was not the intention?

Discussed with @jounikor about this. LTE-M uses the same roaming agreements as legacy LTE, so it's more probable that roaming works with LTE-M than NB-IoT. Anyway, I would like to clarify the table so, that it's not possible to get the impression that roaming does not work with NB-IoT at all. For example, "Better roaming agreements" or something like that.

Author: Actually, I wanted to point to the white paper : https://infocenter.nordicsemi.com/topic/nwp_044/WP/nwp_044/lte_technology.html. Sorry about that.

I will update the table as per the above suggestion.

Some bands use :term:`Frequency Division Duplex (FDD)` and some use :term:`Time Division Duplex (TDD)`.
The different bands are licensed by different Mobile Network Operators (MNO) around the world.
All LTE products run subsets of bands.
Traditional LTE modules support bands ranging from 1 to 5, which are geographic or MNO-specific.
For cellular IoT, lower frequencies provide a longer range.
The half-duplex operation opens for much simpler multiband support and enables :term:`User equipment (UE)` to operate in a larger region.

Bandwidth comparison
--------------------

The following is a LTE bandwidth comparison for LTE-M and NB-IoT:

+----------------------------------+----------------------------------+----------------------------------+
|                                  | LTE-M                            | NB-IoT                           |
+==================================+==================================+==================================+
| Bandwidth                        | Less bandwidth (1.4 MHz)         | Less bandwidth (200KHz)          |
+----------------------------------+----------------------------------+----------------------------------+

> | Modulation Techniques            | No 64QAM                         | No 64QAM or 16QAM                |

Reviewer 3: Actually I noticed you have another row about modulation. Just remove this row and it's fine.

| Antenna Techniques               | No MIMO                          | No MIMO                          |
+----------------------------------+----------------------------------+----------------------------------+
| OFDM subcarriers                 | Less OFDM subcarriers (72)       | OFDM subcarriers (12)            |
+----------------------------------+----------------------------------+----------------------------------+
| Duplexing                        | Can use half-duplex              | Must use half-duplex             |
+----------------------------------+----------------------------------+----------------------------------+
| Modulation in :term:`Uplink (UL)`| QPSK or 16QAM in both UL and DL  | BPSK or QPSK in UL depending on  |
| and :term:`Downlink (DL)`        | depending on signal quality      | signal quality.                  |
|                                  |                                  | Only QPSK in DL                  |
+----------------------------------+----------------------------------+----------------------------------+

Location support
================

> A factor governing the usage of the protocol is the support for the technologies in the areas where you wish to install your product.

Reviewer 3: ```suggestion
A factor governing the selection of the access technology is the MNO support in the areas where you wish to deploy your product.
```

> A factor governing the usage of the protocol is the support for the technologies in the areas where you wish to install your product.

Reviewer: It's better now, although `a factor governing the selection` still sounds a bit complex to me. 😄 

I think the heading could be `Access technology`, `Access technology selection` or something like that instead.

Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.
See `Mobile IoT Deployment Map`_ for more information.

Certifications
==============

> Some MNOs have their own certifications, like Verizon and Vodafone.

Reviewer 3: ```suggestion
Some MNOs, such as Verizon and Vodafone, have their own certification requirements.
```

But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.

For more information about certification related to the nRF91 Series, refer to the following pages:

* `nRF91 Series certifications`_
* `Certifying a Cellular IoT Device`_

SIM card support
================

> SIMs that are used with the nRF91 Series devices must have low-power WAN support and must be supported by the MNO.

Reviewer 3: ```suggestion
SIMs that are used with the nRF91 Series devices must support LTE-M, NB-IoT or both.
```

> For more information on the iBasis SIM compatible with the nRF91 Series devices, refer to the `iBasis IoT network coverage`_ page.

Reviewer 3: ```suggestion
For more information on the iBasis SIM bundled with the nRF91 Series devices, refer to the `iBasis IoT network coverage`_ page.
```

Reviewer 2: Ibasis is only bundled for nRF9160 devices.

Onomondo SIMs are bundled for nRF9161 DK.
lte-m https://onomondo.com/product/coverage/lte-m-networks/
nb-iot https://onomondo.com/product/coverage/nb-iot-networks/

SoftSIM
-------

The nRF91 Series supports SoftSIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
The following are some of the key advantages of using SoftSIM:

> * Cost savings if you remove the hardware.

Reviewer 3: ```suggestion
* Cost savings and simplified design thanks to the omission of SIM hardware layer.
```

* Power savings in idle mode.
  Applications using eDRX will benefit from using SoftSIM.

> * Increases battery life.

Author: Updated the bullet points.

See the SoftSIM support section of the `Cellular IoT unique features`_ documentation for information on SoftSIM.

The SoftSIM support is available in |NCS| from the v2.5.0 release.
The `Onomondo SoftSIM intergration with the nRF91 Series`_ guide describes the intergation of SoftSIM into |NCS|, and the :ref:`nrf_modem_softsim` guide demonstrates the transfer of SIM data between the modem and the application.

Power consumption
=================

LTE-M and NB-IoT both support Power Saving Mode (PSM) or Extended Discontinuous Reception (eDRX) to minimize power consumption.
For more information on power-saving techniques, refer to the DevAcademy's `Power saving techniques`_ documentation.

Security
========

The nRF91 Series devices include a range of security features, such as Arm TrustZone and Arm CryptoCell for secure application and data handling.
Security best practices are impletemented to protect data transmitted over the LTE network.
This includes using encryption, secure boot, and secure firmware updates.

> Getting started with Cellular products

Reviewer 3: I would not capitalize "cellular". If some capitals are desired, consider writing e.g. "nRF cellular products"

**************************************

> For getting started on the cellular products, you can refer to the :ref:`ug_nrf91` documentation, which guides through the features, getting started steps, and advanced usage of the boards.

Reviewer 3: ```suggestion
For getting started with the cellular products, you can refer to the :ref:`ug_nrf91` documentation, which guides through the features, getting started steps, and advanced usage of the boards.
```

> For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_ , which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor’s 91 Series cellular hardware.

Reviewer: Please align the naming for `Nordic Semiconductor’s 91 Series cellular hardware`. I think the terms used in the documentation are:

- nRF91 Series device (nRF91 DKs and Thingy:91s)
- nRF91 Series SiP

> The :ref:`nrf9160_ug_network_mode` section provides informaiton about switching between LTE-M and NB-IoT.

Reviewer: ```suggestion
The :ref:`nrf9160_ug_network_mode` section provides information about switching between LTE-M and NB-IoT.
```

Cellular IoT in |NCS|
*********************

> The following sections explain the different applications, libraries, and sample that use the cellular IoT products.

Reviewer: ```suggestion
The following sections explain the different applications, libraries, and samples that use the cellular IoT products.
```

Modem library
=============

.. include:: ../../device_guides/working_with_nrf/nrf91/nrf91_features.rst
    :start-after: nrf91_modem_lib_start
    :end-before: nrf91_modem_lib_end

Library Support
===============

The following nRF Connect SDK libraries are used for cellular IoT:

* :ref:`lib_modem`
* :ref:`lib_networking`
* :ref:`liblwm2m_carrier_readme` library

> * :ref:`mod_memfault` library

Author: discussed offline and this need not be addressed now.

Applications and samples
========================

The following applications use the cellular IoT in |NCS|:

* :ref:`asset_tracker_v2`
* :ref:`serial_lte_modem`
* :ref:`connectivity_bridge`
* `nRF Asset Tracker project`_

The following samples are used for the cellular IoT development in |NCS|:

* :ref:`cellular_samples`
* :ref:`networking_samples` - These samples use an nRF91 Series device over LTE or an nRF70 Series device over Wi-Fi for communication and connection.
* :ref:`memfault_sample` sample

> Intergrations

Reviewer 4: ```suggestion
Integrations
```

=============

> The following intergrations are available for cellular IoT in |NCS|:

Reviewer 4: ```suggestion
The following integrations are available for cellular IoT in |NCS|:
```

* :ref:`ug_nrf_cloud` - `nRF Cloud`_ is `Nordic Semiconductor's IoT cloud platform`_ that allows you to remotely manage and update your IoT devices using :term:`Firmware Over-the-Air (FOTA) <Firmware Over-the-Air (FOTA) update>`.

> * :ref:`ug_memfault`- Debug issues remotely of cellular IoT devices over LTE.

Reviewer: Maybe this could be rephrased to make it easier to understand.

> * :ref:`ug_memfault`- Debug issues remotely of cellular IoT devices over LTE.

Author: This sentence was rephrased.

|                  | Lower latency                  | Better penetration             |
+                  +--------------------------------+--------------------------------+
|   Strengths      | Roaming                        | Power efficient at lower data  |
|                  |                                | rate                           |
+                  +--------------------------------+--------------------------------+
|                  | Power efficient at medium data |                                |
|                  | rate                           |                                |
+                  +--------------------------------+--------------------------------+
|                  | Suitable for TCP/TLS secure    |                                |
|                  | connection                     |                                |
+------------------+--------------------------------+--------------------------------+

You can find the key feature comparison between LTE-M and NB-IoT in the `nWP044 - Best practices for cellular IoT development`_ documentation.

Factors to be considered for LTE
********************************

> Following are some key aspects to ensure LTE support for your SiP using Nordic Semiconductor's devices, such as the nRF91 Series:

Reviewer 4: ```suggestion
Following are some key aspects to ensure LTE support for your SiP using Nordic Semiconductor's devices, such as the nRF91 Series.
```

LTE bands
=========

There are 56 LTE bands from 400 MHz to 6 GHz.
Some bands use :term:`Frequency Division Duplex (FDD)` and some use :term:`Time Division Duplex (TDD)`.

> The different bands are licensed by different Mobile Network Operators (MNO) around the world.
> All LTE products support a subset of frequency bands.
> Traditional LTE modules support one to five bands.
> This means they are geographic or MNO-specific.
> Half-duplex operation opens for much simpler multiband support and enables :term:`User equipment (UE)` to operate in a larger region.

Reviewer 4: ```suggestion
The different bands are licensed by different Mobile Network Operators (MNO) around the world.

All LTE products support a subset of frequency bands.
Traditional LTE modules support one to five bands.
This means they are geographic or MNO-specific.
Half-duplex operation opens for much simpler multiband support and enables :term:`User equipment (UE)` to operate in a larger region.
```
Just to break the flow a bit.

| and :term:`Downlink (DL)`        | depending on signal quality      | signal quality.                  |
|                                  |                                  | Only QPSK in DL                  |
+----------------------------------+----------------------------------+----------------------------------+

Location support
================

A factor governing the selection of the access technology is the MNO support in the areas where you wish to deploy your product.
Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.
See `Mobile IoT Deployment Map`_ for more information.

Certifications
==============

Some MNOs, such as Verizon and Vodafone, have their own certification requirements.
But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.

For more information about certification related to the nRF91 Series SiP, refer to the following pages:

* `nRF91 Series certifications`_
* `Certifying a Cellular IoT Device`_

SIM card support
================

> SIMs that are used with the nRF91 Series devices must support LTE-M, NB-IoT or both.

Reviewer 4: ```suggestion
SIMs that are used with the nRF91 Series devices must support LTE-M, NB-IoT, or both.
```

The iBasis SIM is bundled with the nRF9160 DK and Thingy:91, and Onomondo SIM is bundled with the nRF9161 DK.
Check the `iBasis IoT network coverage`_, `Onomondo LTE-M coverage`_, or `Onomondo NB-IoT coverage`_ pages to see the network coverage for different countries, depending on the SIM card you are using.

Software SIM
------------

The nRF91 Series supports software SIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
The following are some of the key advantages of using software SIM:

* Cost savings and simplified design by excluding the SIM hardware layer.

> * Power savings especially in idle mode using eDRX.

Reviewer 4: ```suggestion
* Power savings, especially in idle mode using eDRX.
```

See the software SIM support section of the `Cellular IoT unique features`_ documentation for information on software SIM.

> The software SIM support is available in |NCS| from the v2.5.0 release.

Reviewer 4: ```suggestion
The software SIM support is available in the |NCS| from the v2.5.0 release.
```

> The `Onomondo SoftSIM intergration with the nRF91 Series`_ guide describes the intergation of SoftSIM into |NCS|, and the :ref:`nrf_modem_softsim` guide demonstrates the transfer of SIM data between the modem and the application.

Reviewer 5: ```suggestion
The `Onomondo SoftSIM integration with the nRF91 Series`_ guide describes the integration of SoftSIM into |NCS|, and the :ref:`nrf_modem_softsim` guide demonstrates the transfer of SIM data between the modem and the application.
```

Power consumption
=================

LTE-M and NB-IoT both support Power Saving Mode (PSM) or Extended Discontinuous Reception (eDRX) to minimize power consumption.
For more information on power-saving techniques, refer to the DevAcademy's `Power saving techniques`_ documentation.

> Security

Reviewer 4: Would it make sense to mention https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/board_support/processing_environments.html and link to this page here?

Author: May be we could add after first sentence, where we mention about Arm TrustZone ? @tokangas : Could you please confirm.
Then, we are mainly talking about protecting data transmitted over the LTE network.

Reviewer: Yes, it's fine by me.

Security
========

The nRF91 Series devices include a range of security features, such as Arm TrustZone and Arm CryptoCell for secure application and data handling.
Security best practices are also implemented to protect data transmitted over the LTE network.

> This also includes using Transport Layer Security (TLS) for Transmission Control Protocol (TCP) and Datagram Transport Layer Security (DTLS) for User Datagram Protocol (UDP) as security protocols.

Reviewer: The word `also` is present in two sentences, maybe one of them could be removed. The words `security` and `protocol` are repeated several times, not sure if `as security protocols` is really needed. Would the following be enough?
```suggestion
This includes using Transport Layer Security (TLS) for Transmission Control Protocol (TCP) and Datagram Transport Layer Security (DTLS) for User Datagram Protocol (UDP).
```

Getting started with cellular products
**************************************

For getting started with the cellular products, you can refer to the :ref:`ug_nrf91` documentation, which guides through the features, getting started steps, and advanced usage of the boards.
For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_ , which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor’s nRF91 Series device.

> The :ref:`nrf9160_ug_network_mode` section provides information about switching between LTE-M and NB-IoT.

Reviewer 4: Suggestion for later: This might work as a separate page.

Cellular IoT in |NCS|

> *********************

Reviewer 4: Suggestion for later: This might work as a separate page.

The following sections explain the different applications, libraries, and samples that use the cellular IoT products.

Modem library
=============

.. include:: ../../device_guides/working_with_nrf/nrf91/nrf91_features.rst
    :start-after: nrf91_modem_lib_start
    :end-before: nrf91_modem_lib_end

Library support
===============

The following |NCS| libraries are used for cellular IoT:

* :ref:`lib_modem`
* :ref:`lib_networking`
* :ref:`liblwm2m_carrier_readme` library
* :ref:`mod_memfault` library

Applications and samples
========================

The following applications use the cellular IoT in |NCS|:

> * :ref:`asset_tracker_v2`

Reviewer 6: Add the https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/cellular/nrf_cloud_multi_service/README.html sample if it is not already in `cellular_samples`.

Author: Yeah, the sample is in the cellular_samples folder.

* :ref:`serial_lte_modem`
* :ref:`connectivity_bridge`

> * `nRF Asset Tracker project`_

Reviewer 6: This should be moved to `Integrations` because it is not an NCS application.

The following samples are used for the cellular IoT development in |NCS|:

* :ref:`cellular_samples`

> * :ref:`networking_samples` - These samples use an nRF91 Series device over LTE or an nRF70 Series device over Wi-Fi for communication and connection.

Reviewer 5: ```suggestion
* :ref:`networking_samples` - These samples use an nRF91 Series device over LTE or an nRF70 Series device over Wi-Fi® for communication and connection.
```

* :ref:`memfault_sample` sample.

Integrations
============

The following Integrations are available for cellular IoT in |NCS|:

* :ref:`ug_nrf_cloud` - `nRF Cloud`_ is `Nordic Semiconductor's IoT cloud platform`_ that allows you to remotely manage and update your IoT devices using :term:`Firmware Over-the-Air (FOTA) <Firmware Over-the-Air (FOTA) update>`.
* :ref:`ug_memfault`- A cloud-based web application with |NCS| compatibility that monitors devices and allows you to debug issues remotely through an LTE or Wi-Fi network.
* :ref:`ug_avsystem` - Software solution provider that provides a device management platform.

Power optimization
==================

For optimizing the power consumption of your cellular application, you can use the `Online Power Profiler (OPP)`_ tool.
See :ref:`app_power_opt_nRF91` for more information.

References
**********

* `nWP044 - Best practices for cellular IoT development`_ - White paper that describes the guidelines that you need to consider when you start the development for a low power cellular IoT product.

> * `Cellular IoT asset tracking`_.

Reviewer 6: ```suggestion
* `How nRF9160 enables cellular IoT asset tracking`_.
```

+                  +--------------------------------+--------------------------------+
|                  | Power efficient at medium data |                                |
|                  | rate                           |                                |
+                  +--------------------------------+--------------------------------+
|                  | Suitable for TCP/TLS secure    |                                |
|                  | connection                     |                                |
+------------------+--------------------------------+--------------------------------+

You can find the key feature comparison between LTE-M and NB-IoT in the `nWP044 - Best practices for cellular IoT development`_ documentation.

Factors to be considered for LTE
********************************

Following are some key aspects to ensure LTE support for your SiP using Nordic Semiconductor's devices, such as the nRF91 Series.

LTE bands
=========

There are 56 LTE bands from 400 MHz to 6 GHz.
Some bands use :term:`Frequency Division Duplex (FDD)` and some use :term:`Time Division Duplex (TDD)`.

> The different bands are licensed by different Mobile Network Operators (MNO) around the world.

Reviewer 7: Should probably say here that there are no known deployments for other than FDD for M1/NB

All LTE products support a subset of frequency bands.
Traditional LTE modules support one to five bands.

> This means they are geographic or MNO-specific.

Reviewer: This should probably say something about nRF91 Series band support as well.

Physical layer parameters
-------------------------

The following table provides physical layer parameters for LTE-M and NB-IoT:

+----------------------------------+----------------------------------+----------------------------------+
|                                  | LTE-M                            | NB-IoT                           |
+==================================+==================================+==================================+
| Bandwidth                        | 1.4 MHz                          | 200 kHz                          |
+----------------------------------+----------------------------------+----------------------------------+
| Antenna Techniques               | Single antenna                   |  Single antenna                  |
+----------------------------------+----------------------------------+----------------------------------+
| OFDM subcarriers                 |  OFDM subcarriers (72)           | OFDM subcarriers (12)            |
+----------------------------------+----------------------------------+----------------------------------+

> | Duplexing                        | Can use half-duplex              | Must use half-duplex             |

Reviewer 7: Should say here that no network uses other than HD-FDD for M1..

+----------------------------------+----------------------------------+----------------------------------+
| Modulation in :term:`Uplink (UL)`| QPSK or 16QAM in both UL and DL  | BPSK or QPSK in UL depending on  |
| and :term:`Downlink (DL)`        | depending on signal quality      | signal quality.                  |
|                                  |                                  | Only QPSK in DL                  |
+----------------------------------+----------------------------------+----------------------------------+

Network support
================

A factor governing the selection of the access technology is the MNO support in the areas where you wish to deploy your product.
Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.

> See `Mobile IoT Deployment Map`_ for more information.

Reviewer 5: ```suggestion
See `Mobile IoT deployment Map`_ for more information.
```

Certifications
==============

> Some MNOs, such as Verizon and Vodafone, have their own certification requirements.

Reviewer 7: ```suggestion
Some MNOs, such as Verizon and Vodafone, have their own certification requirements. In such case it is adviced to engage with the MNO certification programs and contacts early on to receive certification related requiremetns and understand better the potential certification costs, process and timelines.
```

Author: Updated, please check.

> But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.

Reviewer 7: Suggest a change:
Global Certification Forum) or PTCRB certifications

> But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.

Reviewer 7: ```suggestion
But many operators only require GCF (Global Certification Forum) or PTCRB certifications that devices work as per standard, and regulatory certifications.
```

For more information about certification related to the nRF91 Series SiP, refer to the following pages:

* `nRF91 Series certifications`_
* `Certifying a Cellular IoT Device`_

SIM card support
================

SIMs that are used with the nRF91 Series devices must support LTE-M, NB-IoT, or both.
The iBasis SIM is bundled with the nRF9160 DK and Thingy:91, and Onomondo SIM is bundled with the nRF9161 DK.
Check the `iBasis IoT network coverage`_, `Onomondo LTE-M coverage`_, or `Onomondo NB-IoT coverage`_ pages to see the network coverage for different countries, depending on the SIM card you are using.

Software SIM
------------

The nRF91 Series supports software SIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
The following are some of the key advantages of using software SIM:

* Cost savings and simplified design by excluding the SIM hardware layer.
* Power savings, especially in idle mode using eDRX.

> 

Reviewer 7: Should probably say the fact  here that traditional carriers (not MVNOs like iBasis or Onomondo and alike) will most likely not allow their profiles be provisioned/uploaded into SoftSIMs. One need to use profiles coming from the SoftSIM provides (such as Onomondo).

Author: Updated. Please check.

For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_ , which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor’s nRF91 Series device.

The :ref:`nrf9160_ug_network_mode` section provides information about switching between LTE-M and NB-IoT.

Cellular IoT in |NCS|
*********************

The following sections explain the different applications, libraries, and samples that use the cellular IoT products.

Modem library
=============

.. include:: ../../device_guides/working_with_nrf/nrf91/nrf91_features.rst
    :start-after: nrf91_modem_lib_start
    :end-before: nrf91_modem_lib_end

Library support
===============

The following |NCS| libraries are used for cellular IoT:

* :ref:`lib_modem`
* :ref:`lib_networking`
* :ref:`liblwm2m_carrier_readme` library
* :ref:`mod_memfault` library

Applications and samples
========================

The following applications use the cellular IoT in |NCS|:

* :ref:`asset_tracker_v2`
* :ref:`serial_lte_modem`
* :ref:`connectivity_bridge`

The following samples are used for the cellular IoT development in |NCS|:

> * :ref:`cellular_samples`.
> * :ref:`networking_samples` - These samples use an nRF91 Series device over LTE or an nRF70 Series device over Wi-Fi® for communication and connection.
> * :ref:`memfault_sample` sample.

Reviewer 5: ```suggestion
* :ref:`cellular_samples`.
* :ref:`memfault_sample` sample.
* :ref:`networking_samples` - These samples use an nRF91 Series device over LTE or an nRF70 Series device over Wi-Fi® for communication and connection.
```

Integrations
============

> The following Integrations are available for cellular IoT in |NCS|:

Reviewer 5: ```suggestion
The following integrations are available for cellular IoT in |NCS|:
```

* `nRF Asset Tracker project`_.

> * :ref:`ug_nrf_cloud` - `nRF Cloud`_ is `Nordic Semiconductor's IoT cloud platform`_ that allows you to remotely manage and update your IoT devices using :term:`Firmware Over-the-Air (FOTA) <Firmware Over-the-Air (FOTA) update>`.

Reviewer 5: ```suggestion
* :ref:`nRF Cloud <ug_nrf_cloud>` - `nRF Cloud`_ is `Nordic Semiconductor's IoT cloud platform`_ that allows you to remotely manage and update your IoT devices using :term:`Firmware Over-the-Air (FOTA) <Firmware Over-the-Air (FOTA) update>`.
```

> * :ref:`ug_memfault`- A cloud-based web application with |NCS| compatibility that monitors devices and allows you to debug issues remotely through an LTE or Wi-Fi network.

Reviewer 5: ```suggestion
* :ref:`Memfault <ug_memfault>` - A cloud-based web application with |NCS| compatibility that monitors devices and allows you to debug issues remotely through an LTE or Wi-Fi network.
```

> * :ref:`ug_avsystem` - Software solution provider that provides a device management platform.

Reviewer 5: ```suggestion
* :ref:` AVSystem <ug_avsystem>` - Software solution provider that provides a device management platform.
```

Power optimization
==================

For optimizing the power consumption of your cellular application, you can use the `Online Power Profiler (OPP)`_ tool.
See :ref:`app_power_opt_nRF91` for more information.

References
**********

> * `nWP044 - Best practices for cellular IoT development`_ - White paper that describes the guidelines that you need to consider when you start the development for a low power cellular IoT product.
> * `How nRF9160 enables cellular IoT asset tracking`_.
> * `Accelerate Cellular Product Development`_.

Reviewer 5: ```suggestion
* `nWP044 - Best practices for cellular IoT development`_ whitepaper - Describes the guidelines that you need to consider when you start the development for a low power cellular IoT product.
* `How nRF9160 enables cellular IoT asset tracking`_ webinar.
* `Accelerate cellular product development`_ webinar.
```

For LTE-M and NB-IoT, the only known deployment is FDD.

nRF91 Series SiPs support up to 18 LTE bands for global operation.
For details of LTE band support in an nRF91 Series SiP for LTE-M and NB-IoT, see `nRF9161 Product Specification`_ or `nRF9160 Product Specification`_, depending on the SiP you are using.

Physical layer parameters
-------------------------

The following table provides physical layer parameters for LTE-M and NB-IoT:

+----------------------------------+----------------------------------+----------------------------------+
|                                  | LTE-M                            | NB-IoT                           |
+==================================+==================================+==================================+
| Bandwidth                        | 1.4 MHz                          | 200 kHz                          |
+----------------------------------+----------------------------------+----------------------------------+
| Antenna Techniques               | Single antenna                   |  Single antenna                  |
+----------------------------------+----------------------------------+----------------------------------+

> | OFDM subcarriers                 |  OFDM subcarriers (72)           | OFDM subcarriers (12)            |

Reviewer 3: In my opinion this does not provide useful additional information to the reader of this document. Number of subcarriers is just another way of indicating the available bandwidth which is already mentioned above.

But if you want to keep it, please remove the repetition of "OFDM Subcarriers" (now it's threre 3 times)
